### ISSUE TITLE
Whitelist 

### ISSUE THREAD

**Issue by ookangzheng** [mkb2091/blockconvert#2]
www.freedownloadmanager.org
pbs.twimg.com

**Comment by mkb2091** [comment]
Thanks for spotting the false positive, I've added them to the whitelist

**Comment by mkb2091** [comment]
I'll keep this issue open for future false positives

**Comment by mkb2091** [comment]
I found files.freedownloadmanager.org, and files2.freedownloadmanager.org are blocked, I'm assuming they are needed so I'll also add them to the whitelist

**Comment by ookangzheng** [comment]
Will you consider adding this 2 whitelist
1. https://github.com/ookangzheng/blahdns/blob/master/hosts/whitelist.txt
2. https://github.com/anudeepND/whitelist

**Comment by mkb2091** [comment]
I am already using the second one, and didn't know about the first. I'll add it now

**Comment by mkb2091** [comment]
I just checked, and the first one is under a CC BY-NC 4.0 license, which isn't compatible with GPLv3, so at the moment I can't add it

**Comment by ookangzheng** [comment]
> I just checked, and the first one is under a CC BY-NC 4.0 license, which isn't compatible with GPLv3, so at the moment I can't add it

First whitelist is maintain by me, so you can add into ur project.
Second question, why CC BY-NC4.0 doesn't compatible with GPLv3?
Im using this license is to prevent from someone selling my whitelist without noticing me

**Comment by mkb2091** [comment]
CC BY-NC4.0 prevents anyone from selling the list, and any copys of the list must also have that license. GPLv3 allows selling, just requires that anyone who receives the list(or a product with the list built in), must have the same freedom.

**Comment by mkb2091** [comment]
Thanks for giving permission to use the whitelist, I've added it, but also added the temp files to .gitignore, so anyone who wants to use your list will have to download it from your github repo

**Comment by ookangzheng** [comment]
www.namecheap.com is blocked

**Comment by mkb2091** [comment]
Thanks for spotting it, previously only namecheap.com had been in a whitelist

**Comment by lightswitch05** [comment]
If you ever have issues with one of my lists, please report it directly to me so that we can have a discussions about why I blocked it and if it should remain blocked or not.

**Comment by mkb2091** [comment]
Yeah, fair enough. I'll try to get in touch with any list maintainers when I add a domain in their list to the whitelist
